### ISSUE TITLE
Status of 'access'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#57]
`access` is currently outside the BFO hierarchy, awaiting resolution.

I propose that it be placed under `disposition` and defined as:

`access` = a disposition that inheres in a person by virtue of the fact they bear the ability, right, and permission to approach and use a resource.

FYI: ["access" is goals and targets](https://github.com/SDG-InterfaceOntology/sdgio/wiki/Key-theme:-Access) and used in indicators: 3.b.1, 4.a.1, 5.6.2, 7.1.1, 11.2.1, 14.b.1, 17.6.1.

Further work needs to be done on related terms:
>`ability to access` = The disposition of an environmental system to physically provide a resource to the consumer or user of that resource.
`permission to access` = A disposition of an environmental system to allow an entity to approach and use a resource.
`right of access` = A disposition of a human societal system to recognise and protect, through various means, the ability of an entity to approach and use a resource.

I believe `permission to` and `right to` are roles borne by a person, not dispositions of a system. These roles are granted, literally created, by persons or organizations that bear the requisite authority to do so.

`available for access` (or `accessibility`) is a disposition of a system, whereas `ability to` is a disposition of a person/organism.

Thoughts?
@phismith @pbuttigieg

**Comment by pbuttigieg** [comment]
> access is currently outside the BFO hierarchy, awaiting resolution.

>I propose that it be placed under disposition and defined as:

>access = a disposition that inheres in a person by virtue of the fact they bear the ability, right, and permission to approach and use a resource.


Given the current definition, your revision is an improvement; however, something doesn't sit well.

I think we should recall the [dictionary definition](http://www.merriam-webster.com/dictionary/access) of access:
> a way of being able to use or get something

That "way of being able" is the simultaneous realisations of the dispositions of an accessor and their environment. A key has the disposition to open a lock, but that lock must also have the disposition to be opened by that key and the environment must allow the key to get into the lock in order for there be a way to unlock that door with that key. It's the environmental aspect of "approach" in "approach and use".

A person my have all the appropriate rights, permissions, and abilities, but still not have access to a resource as something in their socio-environmental system prevents it (a corrupt regulatory entity, for example). If we use the definition you propose, I think that this can be overlooked. 

One could argue that "ability" also suggests that the person can surmount any forces impeding access, including such things as corruption. Are we then saying that it doesn't matter how (avoidably) obstructive the system is, as long as people can get around it? I can see this being useful, but I don't think it gels with the spirit of the SDGs. I think we have to put a comment on "ability to access" describing that this ability should not compensate for things like corruption or unjust or illegal obstruction by human agency.

Given the treatment of accessibility in the issue description: 

`access` = a disposition that inheres in a person who bears the ability, right, and permission to approach and use an *accessible* resource.

Thus the resource in question must bear the `accessibility` disposition, meaning its environmental system will not obstruct (beyond some unavoidable threshold which should be described) a bearer of the relevant ability, right, and permission from approaching and using said resource. The fact that it bears a resource disposition covers the 'disposition of some lock to be opened by some key' idea above.

Also, is there any use of access in the SDG domain that does not feature persons? We may need a new class for machines having access to data, for example.

>I believe permission to and right to are roles borne by a person, not dispositions of a system. These roles are granted, literally created, by persons or organizations that bear the requisite authority to do so.

I don't think we're dealing with roles here.

The above sounds right for `ability to access`. The current definition is more suitable for something like `ability to provide`.

However, I think that 'permission to access` and  `right of access` say more about the system. If X has permission to do Y, there is nothing very special about X; however, the regulatory elements in X's (social) environmental system will not have a mandate to prevent X doing Y. Similar thinking for right.

As we're dealing with human constructs in right and permission, we could say that there are no cases OBI:`planned process` instantiated in a particular socio-environmental system which would obstruct some accessor's approach and use of a resource. These planned processes need not be mandated or legal. This would be a way of differentiating between the 'avoidable' and 'unavoidable' impediments to access. 

>available for access (or accessibility) is a disposition of a system, whereas ability to access is a disposition of a person/organism.

This above sounds right for ability, but - as described above - not so much for permission and right. How would you define accessibility?

**Comment by mark-jensen** [comment]
> `access` = a disposition that inheres in a person who bears the ability, right, and permission to approach and use an accessible resource.

Yes, perhaps with the following modification: A disposition that inheres in a person or population who bears the ability, right, and permission to approach and use an accessible resource.

> The fact that it bears a resource disposition 

One small point, though not entirely relevant here: I believe that an entity is a resource because it bears a role. While that thing must also have dispositions and qualities of the right sort to be a resource, it is ultimately a resource because of extrinsic factors. This is compatible with our (ENVOs) treatment of `waste`. 

>Also, is there any use of access in the SDG domain that does not feature persons? We may need a new class for machines having access to data, for example.

Not that I see. But we should mention this possibility where applicable in comments or notes.

>A key has the disposition to open a lock, but that lock must also have the disposition to be opened by that key and the environment must allow the key to get into the lock in order for there be a way to unlock that door with that key.

Perhaps we should add a note about the reciprocal dependence involved in access, between user and resource. Or more precisely, between the ability of the user to access and the accessibility of the resource. 

>A person my have all the appropriate rights, permissions, and abilities, but still not have access to a resource as something in their socio-environmental system prevents it (a corrupt regulatory entity, for example).

In this case of a corrupt regulatory entity, I would say the person has the ability and right, but lacks permission. Granted that view depends on how we define or elucidate ‘right’ and ‘permission’. I think a right is granted and hence protected, by an established and operational socioeconomic political entity. As such, rights are formal and documented, on the side of policy and legal entities. On the other hand, permission relates more to individuals and authorization, regardless of rights, often on a case-by-case basis. Permission can be granted regardless of right. Likewise it can be taken away. Perhaps a group denying permission to access a resource is operating independently of the rights-granting organization, but nonetheless technically a member of it, as with a corrupt agency. The existence of a right should guarantee permission of access for all it pertains too, but in reality the right to access a resource can be denied if permission is removed.

**Comment by mark-jensen** [comment]
>How would you define accessibility?

Working on it and open to suggestions.
